TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx

```diff
@@ -1,18 +1,26 @@
-import './Interactions.scss';
+import classnames from 'classnames/bind';
```

Comment:
👍 This is a really nice refactor. What a joy to review. Down with classes!

Reply:
and nice use of a reducer

Reply:
Thank you 😁 Goodbye classes!